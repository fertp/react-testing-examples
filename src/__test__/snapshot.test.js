import renderer from 'react-test-renderer'
import { render } from '@testing-library/react'

const Header = () => {
  return (
    <header>
      <h1>Soy un header</h1>
    </header>
  )
}


test('Snapshot using render @testing-library', () => {
  const { container } = render( <Header/> )
  expect(container).toMatchSnapshot()
})

test('Snapshot using react-test-renderer', () => {
  const tree = renderer.create(<header>ok</header>).toJSON()
  expect(tree).toMatchInlineSnapshot(`
  <header>
    ok
  </header>
  `)
})