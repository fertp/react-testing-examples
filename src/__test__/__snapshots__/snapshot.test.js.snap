// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot using render @testing-library 1`] = `
<div>
  <header>
    <h1>
      Soy un header
    </h1>
  </header>
</div>
`;
